java/examples/sign-request/src/main/java/com/truelayer/examples/sign_request/Program.java

```diff
@@ -0,0 +1,63 @@
+package com.truelayer.examples.sign_request;
+
+import com.truelayer.signing.Signer;
+import okhttp3.*;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.Random;
+import java.util.UUID;
+
+public class Program {
+
+    public static void main(String[] args) {
+        // Read required env vars
+        String access_token = System.getenv("ACCESS_TOKEN");
+        if (access_token == null) throw new RuntimeException("Missing env var ACCESS_TOKEN");
+
+        String kid = System.getenv("KID");
+        if (kid == null) throw new RuntimeException("Missing env var KID");
+
+        String privateKeyPem = System.getenv("PRIVATE_KEY");
+        if (privateKeyPem == null) throw new RuntimeException("Missing env var PRIVATE_KEY");
+
+        // A random body string is enough for this request as `/test-signature` endpoint does not
+        // require any schema, it simply checks the signature is valid against what's received.
+        byte[] bytes = new byte[7];
+        new Random().nextBytes(bytes);
+        String body = "msg-" + new String(bytes, Charset.defaultCharset());
```

Comment:
Can this be simpler, we just want to show that a random body signed will work. If it's more complicated it looks like it may be part of the signing process somehow

Reply:
+1